mcmc/FitterBase.cpp

```diff
@@ -1134,8 +1133,8 @@ void FitterBase::RunSigmaVar() {
       TH1D ***sigmaArray_y_norm = new TH1D**[numVar]();
 
       // Set up for single mode
-      TH1D ****sigmaArray_mode_x = NULL;
-      TH1D ****sigmaArray_mode_y = NULL;
+      TH1D ****sigmaArray_mode_x = nullptr;
```

Comment:
Can we make this a nested vector (or better an std::unordered_map) instead of pointer array?

Reply:
my laziness... All changes I did were valdiated by CI. This would requeir actual checks